lib/Crypto/EncryptAll.php

```diff
@@ -119,6 +119,33 @@ public function __construct(
 		$this->userPasswords = [];
 	}
 
+	/**
+	 * Call this method only when no master key is created.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return bool true when masterkey and sharekey is created else false
+	 */
+	public function createMasterKey(InputInterface $input, OutputInterface $output) {
+		$this->keyManager->setPublicShareKeyIDAndMasterKeyId();
+		$output->writeln("New masterkey created successfully\n");
```

Comment:
This shouldn't be needed. Both the input and output interfaces aren't needed.
Just return true / false to indicate that the keys have been created and validated, and let the caller write the message.

Reply:
Removed. And moved to the caller ( in this context recreate master key command )